lib/graphql-pundit/instrumenter.rb

```diff
@@ -25,12 +27,15 @@ def instrument(_type, field)
 
       def resolve_proc(current_user, old_resolve, options)
         lambda do |obj, args, ctx|
-          query = options[:query].to_s + '?'
-          record = options[:record] || obj
           begin
-            unless ::Pundit.authorize(ctx[current_user], record, query)
-              raise ::Pundit::NotAuthorizedError
-            end
+            result = if options[:proc]
+                       options[:proc].call(obj, args, ctx)
+                     else
+                       query = options[:query].to_s + '?'
+                       record = options[:record] || obj
```

Comment:
It's there, but not named `record`. It's just the second argument passed to `authorize`